src/styles/embeds/sass/components/buttons.css

```diff
@@ -30,7 +30,7 @@
 
   &:hover,
   &:focus {
-    .product__variant-img {
+    & .product__variant-img {
```

Comment:
ok so this syntax is different from sass right? in sass this would mean `.shopify-buy_btn:focus.product__variant-img` whereas this now means `.shopify-buy__btn:focus .product__variant-img` correct?